Update configure-endpoints-gp.md (#5796)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md

7. Select **Run whether user is logged on or not** and check the **Run with highest privileges** check box.

> 8. In the Name field, type an appropriate name for the scheduled task (e.g., Defender for Endpoint Deployment).

Reviewer: ```suggestion
8. In the Name field, type an appropriate name for the scheduled task (for example, Defender for Endpoint Deployment).
```

> 9. Go to the **Actions** tab and click **New...** Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOnboardingScript.cmd* file.

Reviewer: ```suggestion
9. Go to the **Actions** tab and select **New...** Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOnboardingScript.cmd* file.
```

> 10. Click **OK** and close any open GPMC windows.

Reviewer: ```suggestion
10. Select **OK** and close any open GPMC windows.
```

7. Select **Run whether user is logged on or not** and check the **Run with highest privileges** check-box.

> 8. In the Name field, type an appropriate name for the scheduled task (e.g., Defender for Endpoint Deployment).

Reviewer: ```suggestion
8. In the Name field, type an appropriate name for the scheduled task (for example, Defender for Endpoint Deployment).
```

> 9. Go to the **Actions** tab and click **New...**. Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOffboardingScript_valid_until_YYYY-MM-DD.cmd* file.

Reviewer: ```suggestion
9. Go to the **Actions** tab and select **New...**. Ensure that **Start a program** is selected in the **Action** field. Enter the UNC path, using the file server's fully qualified domain name (FQDN), of the shared *WindowsDefenderATPOffboardingScript_valid_until_YYYY-MM-DD.cmd* file.
```

> 10. Click **OK** and close any open GPMC windows.

Reviewer: ```suggestion
10. Select **OK** and close any open GPMC windows.
```
